Monika After Story/game/script-topics.rpy

```diff
@@ -12030,3 +12030,44 @@ label monika_perspective:
     m 1hksdlb "Well I hope that helped you understand at least a little, [player]."
     m "It's the best way I could try and explain it!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_relationcomfort",
+            category=["romance","advice"],
+            prompt="Comfortable in a relationship",
+            random=True
+        )
+    )
+
+label monika_relationcomfort:
+    m 1wud "You know [player], I've been thinking about our relationship lately..."
+    m 2lssdrb "It didn't really start in a way most of them do, so it's not exactly what I would call normal."
+    m 2esa "But the more I thought about it, the more I realized that in a way it is actually pretty standard."
```

Comment:
Put a comma after "way".

Reply:
Another unnecessary one. :thinking: Which type of comma would this be?
It's a single clause.

Reply:
I guess it's not really necessary. I just read it with a slight pause right there.